Sources/Widgets/Manipulators/AbstractManipulator/index.js

```diff
@@ -27,6 +30,27 @@ function vtkAbstractManipulator(publicAPI, model) {
     if (model.widgetNormal) return model.widgetNormal;
     return [0, 0, 1];
   };
+
+  model._computeDeltaFromPrevCoords = (curWorldCoords) => {
+    if (!model._prevWorldCoords?.length || !curWorldCoords?.length)
+      return [0, 0, 0];
```

Comment:
if _prevWorldCoords is empty/null, shouldn't it return curWorldCoords?
if curWorldCoords is empty/null, shouldn't it return _prevWorldCoords ?

Reply:
I think that if there's no previous world coordinate (e.g. first call), then returning [0,0,0] makes sense, since there's no movement. I can also see the case for returning `curWorldCoords` only in the case where `curWorldCoords` is not null, since it's interpreted as a delta from the origin, but that has a built-in assumption on the first call.

If `curWorldCoords` is null, then we shouldn't return the previous world coordinate. That would invalidate the concept of a delta entirely.